webplugin/js/app/mck-sidebox-1.0.js

```diff
@@ -622,9 +622,12 @@ var KM_ATTACHMENT_V2_SUPPORTED_MIME_TYPES = ["application","text","image"];
                 }  
             }
             if(kommunicateCommons.isObject(WIDGET_SETTINGS) && WIDGET_SETTINGS.popup) {
+                var greetingMsgTone = "0.8";
                 ringToneService = new RingToneService();
+                notificationToneOption={};
+                    notificationToneOption.volume = WIDGET_SETTINGS.greetingMsgTone||greetingMsgTone;
                 try {
-                    mckChatPopupNotificationTone = ringToneService.loadChatPopupTone(MCK_CHAT_POPUP_NOTIFICATION_TONE_LINK);
+                    mckChatPopupNotificationTone = ringToneService.loadChatPopupTone(MCK_CHAT_POPUP_NOTIFICATION_TONE_LINK,notificationToneOption);
```

Comment:
@Kusum-Dhoundiyal Create a separate variable for chatPopupNotification, don't use the variable which is getting used messageNoticficationTone which is "notificationToneOption";